.travis.yml

```diff
@@ -6,8 +6,10 @@ matrix:
       env: NODE_VERSION=v6.10.3
     - os: linux
       env: NODE_VERSION=v8.11.1 TRAVIS_PUBLISH_PACKAGES=true
+    - os: linux
```

Comment:
Can we not add yet another leg?  Is it sufficient to just use the osx leg for 10.7 testing or do we get meaningful coverage from using the same build on *nix as well?